scalableminds/webknossos
# user access query: separate organizations

Remove the rule that a user with no teams can be seen by everyone.
This used to be the workaround for admins to fix broken user accounts.
Now the same thing is already solved since an organizaton’s admin can see all of the organization’s users, no matter their teams.

### Steps to test:
- set up two organizations, check if a user can see another if they have a common team OR if the requesting user is organization admin. never should anyone see a user of a different organization

### Issues:
- fixes #2477

------
